Tests/L0/Maven/_suite.ts

```diff
@@ -1352,21 +1348,115 @@ describe('Maven Suite', function () {
             });
     });
 
-    it('Maven with PMD - Executes PMD and uploads results when multiple modules are present', function (done) {
+    it('Maven with PMD - Skips PMD goals if PMD is not enabled', function (done) {
```

Comment:
This test specifically sets pmdAnalysisEnabled to false. It is no longer a super-useful test, though, no.